dataRetriever/requestHandlers/resolverRequestHandlerFactory.go

```diff
@@ -0,0 +1,30 @@
+package requestHandlers
```

Comment:
optional: we might move these factories in dataRetriever/requestHandlers/factory sub-package

Reply:
will move them in another PR